### PR TITLE
Setup action to check if changelog is updated

### PR DESCRIPTION
**Pull request by arddluma**
Added github action that checks if CHANGELOG.md is updated, if it not updated adds comment as shown in this PR
image

### CODE REVIEW

**Comment by poocart** [comment]
@arddluma where it echoes?

**Comment by arddluma** [comment]
> @arddluma where it echoes?

https://github.com/etherspot/transaction-kit/actions/runs/5132250756/jobs/9233301244?pr=35
Here ^ But I will make changes as per your suggestion

**Review by vnovacev — APPROVED** [review]
lgtm

**Review by poocart — APPROVED** [review]
well done! 🔥  but I would wait until @IAmKio reviews actual message copies, thx!

File: .github/workflows/check-npm-version.yml [hunk 1]
```diff
@@ -7,18 +7,35 @@ on:
...
+        with:
+          files: |
+            CHANGELOG.md
+      - name: Log when package version changed
         if: steps.check.outputs.changed == 'true'
         run: 'echo "Yayy!! Version change found in commit ${{ steps.check.outputs.commit }}! New version: ${{ steps.check.outputs.version }} (${{ steps.check.outputs.type }})"'
-      - name: Log when unchanged
+      - name: Log when package version unchanged
         if: steps.check.outputs.changed == 'false'
         run: echo "No version change :/ Please update version in package.json!" && exit 1
```
Comment (IAmKio) [inline .github/workflows/check-npm-version.yml:32]: Could we also have this check add a PR comment?

**Review by IAmKio — COMMENTED** [review]
Just one minor suggestion but other than that - looks good!

File: .github/workflows/check-npm-version.yml (hunk 1 above)
Comment (arddluma) [reply .github/workflows/check-npm-version.yml:38]: Thanks for the suggestion! Made the changes and the comment is shown below

**Pull request merged**